### PR TITLE
Fix #795 File Watch support

### LINKED ISSUE #795: File Watch support

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: docs/modules/ROOT/pages/extensions/file-watch.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[[netty]]
```
Comment (jamesnetherton) [inline docs/modules/ROOT/pages/extensions/file-watch.adoc]: Not netty :slightly_smiling_face:

Comment (ppalaga) [reply docs/modules/ROOT/pages/extensions/file-watch.adoc]: +1, thanks

**Comment by ppalaga** [comment]
a00ca02 fixes the issue found by @jamesnetherton

**Comment by jamesnetherton** [comment]
One question before approving - what's the difference between the `mac` and `non-mac` profile? I didn't see anything obvious aside from the comment about exclusions.

**Comment by ppalaga** [comment]
> what's the difference between the `mac` and `non-mac` profile?

Good catch, it is a leftover from my earlier experiments where I tried to exclude JNA for non-mac but it did not work well in the native mode.

Removed the profiles in 929889d.

**Comment by ppalaga** [comment]
f83e7b8 removed the profiles correctly.

**Pull request merged**